<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaClasse</class>
 <widget class="QMainWindow" name="MaClasse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaClasse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="homepage">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>200</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>400</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>START</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>230</y>
          <width>42</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>110</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>   TETRIS</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="isPartially">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>230</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Yes </string>
        </property>
       </widget>
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>230</y>
          <width>42</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>200</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Partially Fill Board ?</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="levelSpinBox">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>230</y>
          <width>42</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>200</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>200</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
       <widget class="QComboBox" name="gameTypeComboBox">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>230</y>
          <width>72</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>200</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>gameType</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>160</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="additionalInfos"/>
       </widget>
      </widget>
      <widget class="QWidget" name="gamePage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>981</width>
          <height>601</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>10</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>10</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Tetris</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Next piece : </string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="nextPieceGrid"/>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="TetrisGrid"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Score : </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="ScoreLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Lines : </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="LinesLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Level</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="LevelLabel">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="endPage">
       <widget class="QLabel" name="endMessageLabel">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>110</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>280</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Go back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
